src/bin/solang.rs

```diff
@@ -309,20 +316,22 @@ fn main() {
         };
 
         let opt = Options {
-            dead_storage: !matches.is_present("DEADSTORAGE"),
-            constant_folding: !matches.is_present("CONSTANTFOLDING"),
-            strength_reduce: !matches.is_present("STRENGTHREDUCE"),
-            vector_to_slice: !matches.is_present("VECTORTOSLICE"),
+            dead_storage: *matches.get_one::<bool>("DEADSTORAGE").unwrap(),
+            constant_folding: *matches.get_one::<bool>("CONSTANTFOLDING").unwrap(),
+            strength_reduce: *matches.get_one::<bool>("STRENGTHREDUCE").unwrap(),
+            vector_to_slice: *matches.get_one::<bool>("VECTORTOSLICE").unwrap(),
```

Comment:
Doesn't removing the exclamation mark change the behavior of this?

Reply:
Yes, it does, but the option has been inverted here https://github.com/hyperledger-labs/solang/pull/871/files#diff-72099b82607818ec32d54c033604bf812e6a397f78ca1af0180295757cdc79d2R162

Reply:
The new line `.action(ArgAction::SetFalse)`